### DISCUSSION TITLE
how to associate interfaces with VLAN's ?

### DISCUSSION THREAD

**Discussion by lanman9000** [netbox-community/netbox#14997] [Help Wanted!] [answered]
Maybe I've never noticed it before, or its a new feature introduced in the last little bit, but I notice that when you click on an interface for a device or VM, there is a section below the IP addresses called VLANs. 

How do I associate a VLAN to the interface? I've tried looking at the VLAN, the interface, the IP address, and the prefix .. but I don't see any options in any of those to associate together. 

How is this accomplished?
![vlans](https://github.com/netbox-community/netbox/assets/158310803/35bd7e11-27da-45e4-a436-27311927b662)

**Comment by candlerb** [comment, marked as answer]
Click on the "Edit" button (yellow, top right). Then scroll down to the section headed "802.1Q Switching". There you can select one untagged VLAN, and multiple tagged VLANs.

> I've tried looking at the VLAN, the interface, the IP address, and the prefix

You can also associate a prefix with a VLAN (just edit the prefix and select the VLAN), but that's a separate thing and is not related to interfaces.

**Reply by lanman9000** [reply]
oohhh. 802.1q switching. 
Got it, thanks!
